lib/trento_web/router.ex

```diff
@@ -146,7 +146,7 @@ defmodule TrentoWeb.Router do
 
       get "/hosts/:id/exporters_status", PrometheusController, :exporters_status
 
-      get "/software_updates/settings", SoftwareUpdatesController, :get_software_updates_settings
+      resources "/settings/suma_credentials", SoftwareUpdatesController, only: [:index]
```

Comment:
I believe we might want to look at `singleton-resources` https://hexdocs.pm/phoenix/Phoenix.Router.html#resources/4-singleton-resources

Seems to better fit our usecase.

```suggestion
      resources "/settings/suma_credentials", SoftwareUpdatesController, only: [:show], singleton: true
```

Reply:
Let's also rename the controller and file to something like `SoftwareUpdatesCredentialsController` or even `SUMACredentialsController`.

It'd be clearer which resource the actions `show`, `update`, `delete` would be referring to